config/jobs/kubernetes-sigs/blobfuse-csi-driver/blobfuse-csi-driver-config.yaml

```diff
@@ -109,32 +109,33 @@ presubmits:
     spec:
       containers:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20200206-f88edef-master
-        command:
-        - runner.sh
-        - kubetest
         args:
-        # Generic e2e test args
-        - --test
-        - --up
-        - --down
-        - --build=bazel
-        # Azure-specific test args
-        - --provider=skeleton
-        - --deployment=aksengine
-        - --aksengine-agentpoolcount=2
-        - --aksengine-admin-username=azureuser
-        - --aksengine-creds=$(AZURE_CREDENTIALS)
-        - --aksengine-orchestratorRelease=1.16
-        - --aksengine-mastervmsize=Standard_DS2_v2
-        - --aksengine-agentvmsize=Standard_D4s_v3
-        - --aksengine-location=eastus2
-        - --aksengine-public-key=$(AZURE_SSH_PUBLIC_KEY_FILE)
-        - --aksengine-template-url=https://raw.githubusercontent.com/kubernetes-sigs/blobfuse-csi-driver/master/test/manifest/linux.json
-        - --aksengine-download-url=https://github.com/Azure/aks-engine/releases/download/v0.39.1/aks-engine-v0.39.1-linux-amd64.tar.gz
```

Comment:
We are in the process of moving away from legacy job configuration and migrating to PodUtils (see https://github.com/kubernetes/test-infra/issues/16132). Let's just remove `--build=bazel` since building K8s isn't required in this situation and modify necessary flags (e.g. `--aksengine-download-url`, `--aksengine-template-url`, `--aksengine-orchestratorRelease`).

Reply:
@chewong I copied the json content from azure disk driver, and `--build=bazel` is already removed in this PR, any comments? Thanks.

Reply:
Azure Disk and Azure File CSI drivers are still using the legacy job configuration. Ideally, we should run e2e test through `kubetest` like:
```yaml
...
command:
  - runner.sh
    kubetest
args:
  - —up
  - —test
...
```

Reply:
Azure Disk and Azure File CSI Drivers are in the process of migrating to this new configuration format.

Reply:
ok, I see, thanks.

Reply:
fixed, PTAL.